tests/CMakeLists.txt

```
@@ -186,7 +186,10 @@ configure_file(CTestCustom.cmake ${CMAKE_BINARY_DIR}/CTestCustom.cmake)
 
 find_package(Threads REQUIRED)
 add_executable(unittest unittest.cpp frame.cpp)
-target_link_libraries(unittest PUBLIC TestDataModel PRIVATE Catch2::Catch2WithMain Threads::Threads)
+target_link_libraries(unittest PUBLIC TestDataModel PRIVATE Catch2::Catch2WithMain Threads::Threads podio::podioRootIO)
+if (ENABLE_SIO)
+  target_link_libraries(unittest PUBLIC TestDataModel PRIVATE Catch2::Catch2WithMain Threads::Threads podio::podioRootIO SIO::sio podio::podioSioIO)
```

```suggestion
  target_link_libraries(unittest PRIVATE podio::podioSioIO)
```
I think this should pull in all the necessary dependencies, without repeating things that are already there from the call above